modules/migration-rsync-manually-overriding.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: PROCEDURE
+[id="migration-rsync-manually-overriding_{context}"]
+= Manually overriding default non-root operation for data transfer
+
+Although running Rsync pods as non-root user works in most cases, data transfer might fail when when you run workloads as root user on the source side. {mtc-short} provides two ways to manually override default non-root operation for data transfer:
+
+* Configure all migrations to run an Rsync pod as root on the destination cluster for all migrations.
+* Run an Rsync pod as root on the destination cluster per migration.
+
+In both cases, you must set the following labels on the source side of any namespaces that are running workloads with higher privileges prior to migration: `enforce`, `audit`, and `warn.`
+
+.Procedure
+
+* Prior to migration, if you are running workloads as root on the source side of a migration, set the `enforce`, `audit`, and `warn` labels on the source cluster namespaces of the workloads:
++
+[source,terminal]
+----
+oc label pod <pod_name> <label_name>=<label_value>
+----
++
+using the following labels and values:
++
+[source,yaml]
+----
+pod-security.kubernetes.io/enforce: privileged
```

Comment:
IIRC running a workload as root still puts you on the baseline level so setting the labels to "privileged" would actually not be advised.